<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MatchUI</class>
 <widget class="QWidget" name="MatchUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="createMatchGroupBox">
     <property name="title">
      <string>Create Match</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="player1Label">
        <property name="text">
         <string>Player 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="player1ComboBox"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="player2Label">
        <property name="text">
         <string>Player 2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QComboBox" name="player2ComboBox"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="score1Label">
        <property name="text">
         <string>Score 1:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QSpinBox" name="score1SpinBox"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="score2Label">
        <property name="text">
         <string>Score 2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QSpinBox" name="score2SpinBox"/>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="createMatchButton">
        <property name="text">
         <string>Create Match</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="matchTableView"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="deleteMatchButton">
       <property name="text">
        <string>Delete Match</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="updateMatchButton">
       <property name="text">
        <string>Update Match</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="matchDetailsGroupBox">
     <property name="title">
      <string>Match Details</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="player1NameLabel_2">
        <property name="text">
         <string>Player 1:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="player1NameLabel"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="player2NameLabel_2">
        <property name="text">
         <string>Player 2:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLabel" name="player2NameLabel"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="player1ScoreLabel_2">
        <property name="text">
         <string>Score 1:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLabel" name="player1ScoreLabel"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="player2ScoreLabel_2">
        <property name="text">
         <string>Score 2:</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLabel" name="player2ScoreLabel"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>